Gherkin error: defineParameterType is not defined

#### What are you trying to achieve?
I use TS + Playwright + Gherkin. When I try to add defineParameterType I just getting errors.
If I use like in the [documentation](https://codecept.io/bdd/#advanced-gherkin):
```ts
DefineParameterType({
  name: 'popup_type',
  regexp: /critical|non-critical/,
  transformer: (match) => {
    return match === 'critical' ? '[class$="error"]' 
    : '[class$="warning"]';
  },
};);

Given('I see {popup_type} popup', (popup) => {
  I.seeElement(popup);
});
```
The error is :
```bash
⨯ Unable to compile TypeScript:
src/step_definitions/parameterTypes.ts:3:1 - error TS2552: Cannot find name 'DefineParameterType'. Did you mean 'defineParameterType'?
```
Because in `node_modules/codeceptjs/typings/index.d.ts` there is declaration:
```ts
declare function defineParameterType(options: CodeceptJS.IParameterTypeDefinition<any>): void
```
If I use like this:
```ts
defineParameterType({
  name: 'popup_type',
  regexp: /critical|non-critical/,
  transformer: (match) => {
    return match === 'critical' ? '[class$="error"]' 
    : '[class$="warning"]';
  },
};);

Given('I see {popup_type} popup', (popup) => {
  I.seeElement(popup);
});
```
The error is :
```bash
defineParameterType is not defined
ReferenceError: defineParameterType is not defined
```
And if I change in `node_modules/codeceptjs/typings/index.d.ts` from:
```ts
declare function defineParameterType(options: CodeceptJS.IParameterTypeDefinition<any>): void
```
to:
```ts
declare function DefineParameterType(options: CodeceptJS.IParameterTypeDefinition<any>): void
```
and use it like in the [documentation](https://codecept.io/bdd/#advanced-gherkin) everything is working properly.


#### What do you get instead?

Correctly working `defineParameterType` function.

### Details

* CodeceptJS version: 3.6.7
* NodeJS Version: 22.11.0
* Operating System: doesn't matter
* Playwright version: 1.45.0
* Configuration file:

```js

export const config: CodeceptJS.MainConfig = {
  tests: '',
  output: './output',
  helpers: {
    Playwright: {
      browser: 'chromium',
      show: false,
      waitForNavigation: 'load',
      fullPageScreenshots: true,
      waitForTimeout: 15000
    }
  },
  gherkin: {
    features: './src/features/**/*.feature',
    steps: './src/step_definitions/*.ts'
  },
  include: {},
  require: ['ts-node/register'],
  fullPromiseBased: true
};

--- a/typings/index.d.ts
+++ b/typings/index.d.ts
@@ -554,7 +554,7 @@ declare const xScenario: CodeceptJS.IScenario;
 declare const xFeature: CodeceptJS.IFeature;
 declare function Data(data: any): CodeceptJS.IData;
 declare function xData(data: any): CodeceptJS.IData;
-declare function defineParameterType(options: CodeceptJS.IParameterTypeDefinition<any>): void
+declare function DefineParameterType(options: CodeceptJS.IParameterTypeDefinition<any>): void
 
 // Hooks
 declare const BeforeSuite: CodeceptJS.IHook;

--- a/typings/index.d.ts
+++ b/typings/index.d.ts
@@ -599,7 +599,7 @@ declare namespace NodeJS {
     Given: typeof Given;
     When: typeof When;
     Then: typeof Then;
-    DefineParameterType: typeof defineParameterType
+    DefineParameterType: typeof DefineParameterType
   }
 }
 

